app/pages/capability-selector/capabilitySelectorPageContext.js

```diff
@@ -0,0 +1,20 @@
+import content from './manifest.json';
```

Comment:
😨 should we change the name of these files to `capabilities` too?

Reply:
the reason I didn't is because it isn't actually a capabilities page

Reply:
I meant as in `capabilitiesSelectorPageContext`